charts/perses/values.yaml

```diff
@@ -183,15 +183,13 @@ ingress:
 # -- Configure datasources
 # ref: https://github.com/perses/perses/blob/90beed356243208f14cf2249bebb6f6222cb77ae/docs/datasource.md
 datasources:
-  - kind: Datasource
-    metadata:
-      name: PrometheusLocalhost
-      project: perses
-    spec:
-      default: true
-      plugin:
-        kind: PrometheusDatasource
-        spec:
-          directUrl: http://localhost:9090
-          scrapeInterval: "15s"
- 
+  []
+  # - kind: GlobalDatasource
+  #   metadata:
+  #     name: PrometheusDemoBrowser
+  #   spec:
+  #     default: true
+  #     plugin:
+  #       kind: PrometheusDatasource
+  #       spec:
+  #         directUrl: https://prometheus.demo.do.prometheus.io
```

Comment:
@Nexucis I've changed this back to the original content to introduce fewer changes with the PR.